missioncontrol/commands/acquirelicense.go

```diff
@@ -0,0 +1,60 @@
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/jfrog/jfrog-cli-go/missioncontrol/utils"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	"github.com/jfrog/jfrog-cli-go/utils/config"
+	"github.com/jfrog/jfrog-client-go/httpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+func AcquireLicense(bucketId string, name string, mcDetails *config.MissionControlDetails) error {
+	postContent := AcquireLicenseRequestContent{
+		Name:         name,
+		LicenseCount: 1,
+	}
+	requestContent, err := json.Marshal(postContent)
+	if err != nil {
+		return errorutils.CheckError(errors.New("Failed to marshal json. " + cliutils.GetDocumentationMessage()))
+	}
+	missionControlUrl := mcDetails.Url + "api/v1/buckets/" + bucketId + "/acquire"
+	httpClientDetails := utils.GetMissionControlHttpClientDetails(mcDetails)
+	client, err := httpclient.ClientBuilder().Build()
+	if err != nil {
+		return err
+	}
+	resp, body, err := client.SendPost(missionControlUrl, requestContent, httpClientDetails)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != http.StatusOK {
+		return errorutils.CheckError(errors.New(resp.Status + ". " + utils.ReadMissionControlHttpMessage(body)))
+	}
+	log.Debug("Mission Control response: " + resp.Status)
+
+	// Extract license from response
+	var licenseKeys licenseKeys
+	err = json.Unmarshal(body, &licenseKeys)
+	if err != nil {
+		return errorutils.CheckError(err)
+	}
+	if len(licenseKeys.LicenseKeys) < 1 {
+		return errorutils.CheckError(errors.New("failed to acquire license key from mission control: received 0 license keys"))
```

Comment:
failed --> Failed
mission control --> Mission Control

Reply:
Fixed Mission Control. About a capital letter in the beginning of an error message go suggests:
"Error string should not be capitalized or end with punctuation"